README.md

```diff
@@ -1 +1,338 @@
 ![travis ci](https://travis-ci.org/electrode-io/react-native-electrode-bridge.svg?branch=master)
+
+
+Electrode React Native Bridge
+-----------------------------
+
+This project is essentially a react native library, consisting of a JavaScript module and an associated Android Native Module and iOS Native module.
+
+It is built on top of the react native built-in bridging constructs (to communicate between the react native JS side and Native side) and offers a clean bi-directional communication API, exposing methods to send events and/or requests from/to any side of the bridge (JS/Native).
+It offers more options and flexibility to communicate between the JS/Native side that is not offered out of the box by react native. Ultimately it can help with integrating react native applications into existing native code bases.
+It might be used as one of the basic building block of react native apps and native modules.
+
+Here is a non-exhaustive list of a few reasons to use this library as the low level communication bridge instead of the built-in react native constructs :
```

Comment:
It offers more options and flexibility to communicate between the JS/Native side that is not offered out of the box by react native. -> Features offered by `react-native-electrode-bridge` over built-in react native constructs.